oxidation/crates/parsec_api_types/tests/test_addr.rs

```diff
@@ -158,14 +158,6 @@ fn all_addr(testbed: &dyn Testbed) {}
 )]
 fn addr_with_org(testbed: &dyn Testbed) {}
 
-#[template]
```

Comment:
This is not related to this PR, why removing ?

Reply:
Clippy complains about that